### PR TITLE
if CLOSE_TAG is after brace`{`, do not indent

### PR DESCRIPTION
**Pull request by junstyle**
most people wanted:
```
				<?php foreach (array_reverse($ancestors) as $pcat) {?>
				<li><a href="/<?=$pcat->url?>-1"><?=$pcat->name?></a></li>
				<?php }?>
				<li>
					<?=$mdata->name?>
				</li>
```
if do indent, formatted like this:
```
				<?php foreach (array_reverse($ancestors) as $pcat) {
		?>
				<li><a href="/<?=$pcat->url?>-1"><?=$pcat->name?></a></li>
				<?php
	}?>
				<li>
					<?=$mdata->name?>
				</li>
```
so do not indent if CLOSE_TAG is behind brace '{'

### CODE REVIEW

**Comment by ntzm** [comment]
To prove that this new feature works, and doesn't regress in the future, if you could add some tests that would be great

File: src/Fixer/Basic/BracesFixer.php [hunk 1]
```diff
@@ -341,7 +341,12 @@ static function ($item) {
             if (!$startBraceToken->equals('{')) {
                 continue;
             }
-
+            
+            // if CLOSE_TAG is after brace`{`, do not indent
+            if ($tokens[$tokens->getNextNonWhitespace($startBraceIndex)]->getId() == T_CLOSE_TAG) {
```
[the remark this answers is not in the document]
Comment (ntzm) [reply src/Fixer/Basic/BracesFixer.php]: Ah you've done it an even better way, nice one

**Comment by ntzm** [comment]
@junstyle any updates on adding tests to this PR? Thanks

**Comment by SpacePossum** [comment]
closing in favor of https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4339 as that one has unit tests

**Pull request closed without merging**